remix.config.js

```diff
@@ -12,6 +12,10 @@ const baseConfig =
 module.exports = {
   ...baseConfig,
   ignoredRouteFiles: ["**/.*"],
+  // See https://remix.run/docs/en/main/file-conventions/route-files-v2
+  future: {
```

Comment:
Pulling this is so that the new routing works for the Remix Netlify Functions template.